coderadar-ui/src/app/view/configure-project/configure-project.component.ts

```diff
@@ -130,6 +134,27 @@ export class ConfigureProjectComponent implements OnInit {
       });
   }
 
+
+  /**
+   * Gets all of the configured file patterns for the current project and saves them in this.filePatterns.
```

Comment:
The comments on this method still refer to file patterns

Reply:
Done.